numba_dppy/compiler.py

```diff
@@ -34,6 +33,10 @@
 from numba_dppy import config
 from numba_dppy.driver import USMNdArrayType
 from numba_dppy.dppy_array_type import DPPYArray
+from numba_dppy.utils.array_utils import (
+    _is_device_accessible_array,
```

Comment:
What is the correct Pythonic style? If these functions are public functions in `array_utils` they should not have a leading `_`.